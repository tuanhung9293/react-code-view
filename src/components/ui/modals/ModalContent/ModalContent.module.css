.base {
  box-sizing: border-box;
  position: relative;
  width: 600px;
  max-width: 100%;
  padding: 20px;
}

.center {
  text-align: center;
}