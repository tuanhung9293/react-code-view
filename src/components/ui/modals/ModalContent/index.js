import React from 'react'
import classNames from 'classnames'
import styles from './ModalContent.module.css'

function ModalContent({center, ...props}) {
  const className = classNames({
    [styles.base]: true,
    [styles.center]: center
  })

  return <div {...props} className={className} />
}

export default ModalContent