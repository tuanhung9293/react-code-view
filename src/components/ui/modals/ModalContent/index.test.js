import React from 'react'
import {shallow} from 'enzyme'
import ModalContent from './'

describe('rendering', () => {
  it('renders ModalContent with no props', () => {
    expect(shallow(
      <ModalContent />
    )).toMatchSnapshot()
  })

  it('renders ModalContent with props center', () => {
    expect(shallow(
      <ModalContent center />
    )).toMatchSnapshot()
  })

  it('ModalContent should pass props down', () => {
    expect(shallow(
      <ModalContent data="data" data2="data2" />
    )).toMatchSnapshot()
  })
})