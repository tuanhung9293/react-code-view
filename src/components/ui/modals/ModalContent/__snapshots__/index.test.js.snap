// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering ModalContent should pass props down 1`] = `
<div
  className="base"
  data="data"
  data2="data2"
/>
`;

exports[`rendering renders ModalContent with no props 1`] = `
<div
  className="base"
/>
`;

exports[`rendering renders ModalContent with props center 1`] = `
<div
  className="base center"
/>
`;
